PostHogTests/PostHogContextTest.swift

```diff
@@ -50,10 +50,6 @@ class PostHogContextTest: QuickSpec {
 
             let context = sut.dynamicContext()
 
-            #if os(iOS) || os(tvOS)
-                expect(context["$screen_width"] as? Float) != nil
-                expect(context["$screen_height"] as? Float) != nil
-            #endif
```

Comment:
why did we remove this?

Reply:
if this is `nil`, the tests would always fail and CI is happy right now, so I don't follow?

Reply:
True, I think I had issues with simulators back then and have gone with a purely swift test, but it seems like the time to do it if many tests depend on the simulator infrastructure, create an issue?